crates/runtime/tests/runtime.rs

```diff
@@ -426,8 +443,11 @@ fn new_runtime(
         .expect("failed to construct runtime")
 }
 
-async fn run(wasm: impl AsRef<Path>) -> anyhow::Result<Vec<(logging::Level, String, String)>> {
-    const BODY: &str = r#"{"min":42,"max":4242,"port":42424}"#;
+async fn run(
+    wasm: impl AsRef<Path>,
+    is_module: bool,
```

Comment:
Maybe you could use [`wasmparser::is_core_wasm`](https://docs.rs/wasmparser/latest/wasmparser/struct.Parser.html#method.is_core_wasm) here?

Reply:
I forgot that existed, but since this is a test, it is probably easier to specify a bool directly. I'll give it a try when I push any other updates from code review